tests/api-watch-diff/test.ml

```diff
@@ -125,6 +125,14 @@ let val_g_sig =
 
 let ref_signature = [ t_sig; unused_type_sig; val_f_sig ]
 
+let%expect_test "same_signature_test" =
```

Comment:
You can write any string here so no need to use `_` you can simply use spaces instead.

Also adding at the `test` is a bit redundant given we are in a test file, inside a `let%expect_test` so you can remove those suffixes as well.

Reply:
Done. I did it for all the tests.